### PR TITLE
Update uses of text that fail contrast (mostly Reader)

### PR DESCRIPTION
**Pull request by drw158**
#### Changes proposed in this Pull Request

This PR aims to up the contrast for several locations that fail. These instances don't use the text variables as they should. **I avoided anything that might be an icon or something that might not be against a white background.** My goal was to only address text. I'm not too co

I performed 3 searches/updates:

- **Blocks** folder: 
    - replaced `var( --color-neutral-light )` with `var( --color-text-subtle)`
    - replaced `var( --color-neutral-600 )` with `var( --color-text)` 
    - replaced `var( --color-neutral-200 )` with `var( --color-text-subtle)`
- **client/reader folder**: 
    - replaced `var( --color-neutral-light )` with `var( --color-text-subtle)`
- [Updated](https://github.com/Automattic/wp-calypso/pull/32284/files#diff-ae6da2cd3efe0b8333294aea604b19b0L21) a less frequently used extend called `content-font`, mostly used by the Reader. The color of that extend now uses --color-text.

#### Testing instructions

Sorry for the big PR. I don't think this needs involved testing other than a smoke test of the [Blocks](http://calypso.localhost:3000/devdocs/blocks) page and the Reader itself.

Make sure all text is readable and there are no unintended effects.

Before|After
---|---
image|image

### CODE REVIEW

File: client/blocks/author-compact-profile/style.scss [hunk 1]
```diff
@@ -88,7 +88,7 @@
 }
 
 .author-compact-profile__follow-count {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/author-compact-profile/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/author-selector/style.scss [hunk 2]
```diff
@@ -72,5 +72,5 @@
 	width: 168px;
 	font-size: 14px;
 	font-style: italic;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/author-selector/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comment-button/style.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
 .comment-button {
 	align-items: center;
 	box-sizing: border-box;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comment-button/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/comment-actions.scss [hunk 4]
```diff
@@ -1,13 +1,13 @@
 // Actions for Individual Comments
 .comments__comment-actions {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/comment-actions.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/comment-actions.scss [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 	font-size: 14px;
 	list-style: none;
 	margin-top: -6px;
 
 	button {
 		display: inline-block;
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/comment-actions.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/comment-count.scss [hunk 6]
```diff
@@ -1,5 +1,5 @@
 .comments__comment-count {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/comment-count.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/comment-edit-form.scss [hunk 7]
```diff
@@ -57,7 +57,7 @@
 		font-size: 12px;
 		font-weight: 600;
 		text-transform: uppercase;
-		color: var( --color-neutral-200 );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/comment-edit-form.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/form.scss [hunk 8]
```diff
@@ -65,7 +65,7 @@
 		font-size: 12px;
 		font-weight: 600;
 		text-transform: uppercase;
-		color: var( --color-neutral-200 );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/form.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/form.scss [hunk 9]
```diff
@@ -90,7 +90,7 @@
 }
 
 .comments__form-closed {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/form.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/post-comment-list.scss [hunk 10]
```diff
@@ -25,7 +25,7 @@
 }
 
 .comments__view-more {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/post-comment-list.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/post-comment.scss [hunk 11]
```diff
@@ -120,7 +120,7 @@ a.comments__comment-username {
 }
 
 .comments__comment-respondee {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/post-comment.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/post-comment.scss [hunk 12]
```diff
@@ -132,7 +132,7 @@ a.comments__comment-username {
 }
 
 .comments__comment-respondee .comments__comment-respondee-link {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/post-comment.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/post-comment.scss [hunk 13]
```diff
@@ -142,7 +142,7 @@ a.comments__comment-username {
 }
 
 .comments__comment-timestamp a {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/post-comment.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/comments/post-comment.scss [hunk 14]
```diff
@@ -152,7 +152,7 @@ a.comments__comment-username {
 }
 
 .comments__comment-moderation {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/comments/post-comment.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/dismissible-card/style.scss [hunk 15]
```diff
@@ -2,6 +2,6 @@
 	position: absolute;
 	top: 16px;
 	right: 16px;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/dismissible-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/ecommerce-manage-nudge/style.scss [hunk 16]
```diff
@@ -11,7 +11,7 @@
 	position: absolute;
 	top: 14px;
 	right: 18px;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/ecommerce-manage-nudge/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/image-selector/style.scss [hunk 17]
```diff
@@ -162,10 +162,10 @@
 	width: 24px;
 	height: 24px;
 	margin: 0;
-	color: var( --color-neutral-200 );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/image-selector/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/image-selector/style.scss [hunk 18]
```diff
@@ -162,10 +162,10 @@
...
 
 	&:hover {
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/image-selector/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/inline-help/style.scss [hunk 19]
```diff
@@ -435,7 +435,7 @@
 	padding: 16px 0 0;
 	margin: 0;
 	font-size: 14px;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/inline-help/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/like-button/style.scss [hunk 20]
```diff
@@ -4,7 +4,7 @@
 	align-items: center;
 	display: inline-flex;
 	padding: 4px;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/like-button/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/login/style.scss [hunk 21]
```diff
@@ -88,7 +88,7 @@
 		a,
 		a:hover {
 			pointer-events: none;
-			color: var( --color-neutral-light );
+			color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/login/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/login/style.scss [hunk 22]
```diff
@@ -184,7 +184,7 @@
 
 	span {
 		background: var( --color-white );
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/login/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/nps-survey/style.scss [hunk 23]
```diff
@@ -113,5 +113,5 @@
 	top: 0;
 }
 .nps-survey__close-button.button.is-borderless {
-	color: var( --color-neutral-200 );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/nps-survey/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/post-edit-button/style.scss [hunk 24]
```diff
@@ -1,7 +1,7 @@
 .post-edit-button {
 	align-items: center;
 	box-sizing: border-box;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/post-edit-button/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/post-edit-button/style.scss [hunk 25]
```diff
@@ -27,7 +27,7 @@
 	}
 
 	.post-edit-button__label {
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/post-edit-button/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/post-item/style.scss [hunk 26]
```diff
@@ -46,7 +46,7 @@ $expanded-post-item-outline: 4px solid $post-item-border-color;
 	cursor: pointer;
 
 	&:hover .form-checkbox {
-		border-color: var( --color-neutral-200 );
+		border-color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/post-item/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/post-item/style.scss [hunk 27]
```diff
@@ -97,7 +97,7 @@ a.post-item__title-link:visited {
 	}
 
 	.post-item__panel.is-untitled & {
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/post-item/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-author-link/style.scss [hunk 28]
```diff
@@ -1,5 +1,5 @@
 .reader-author-link {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-author-link/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-author-link/style.scss [hunk 29]
```diff
@@ -8,5 +8,5 @@
 }
 
 .reader-author-link:focus {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-author-link/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-combined-card/style.scss [hunk 30]
```diff
@@ -36,7 +36,7 @@
 }
 
 .reader-combined-card__header-post-count {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-combined-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-combined-card/style.scss [hunk 31]
```diff
@@ -173,7 +173,7 @@
 
 .reader-combined-card__post-author-and-time {
 	font-size: 13px;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-combined-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-combined-card/style.scss [hunk 32]
```diff
@@ -194,13 +194,13 @@
 .reader-combined-card .reader-visit-link,
 .reader-combined-card .reader-author-link,
 .reader-combined-card__timestamp-link {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-combined-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-combined-card/style.scss [hunk 33]
```diff
@@ -194,13 +194,13 @@
...
 	cursor: pointer;
 
 	&:focus,
 	&:link,
 	&:visited {
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-combined-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-export-button/style.scss [hunk 34]
```diff
@@ -19,6 +19,6 @@
 
 .reader-export-button__label {
 	font-size: 14px;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-export-button/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-feed-header/style.scss [hunk 35]
```diff
@@ -15,7 +15,7 @@
 }
 
 .reader-feed-header .reader-feed-header__site-badge {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-feed-header/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-feed-header/style.scss [hunk 36]
```diff
@@ -86,7 +86,7 @@
 	}
 
 	.reader-feed-header__follow-count {
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-feed-header/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

Comment (drw158) [inline client/blocks/reader-feed-header/style.scss]:
```suggestion
		color: var( --color-text-subtle );
```

File: client/blocks/reader-full-post/style.scss [hunk 37]
```diff
@@ -469,7 +469,7 @@
 
 .reader-full-post__header-date-link,
 .reader-full-post__header-date-link:visited {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-full-post/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-full-post/style.scss [hunk 38]
```diff
@@ -494,7 +494,7 @@
 }
 
 .reader-full-post__header-tag-list {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-full-post/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/upload-image/style.scss [hunk 39]
```diff
@@ -101,7 +101,7 @@
 	width: 20px;
 	height: 20px;
 	margin: 0;
-	color: var( --color-neutral-200 );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/upload-image/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/term-tree-selector/style.scss [hunk 40]
```diff
@@ -102,7 +102,7 @@ input[type=checkbox].term-tree-selector__input {
 	margin-left: 8px;
 	font-size: 11px;
 	text-transform: uppercase;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/term-tree-selector/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/taxonomy-manager/style.scss [hunk 41]
```diff
@@ -103,7 +103,7 @@
 		.ellipsis-menu__toggle.button.is-borderless {
 			cursor: pointer;
 			font-size: 24px;
-			color: var( --color-neutral-light );
+			color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/taxonomy-manager/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/subscription-length-picker/style.scss [hunk 42]
```diff
@@ -102,7 +102,7 @@
 
 	.subscription-length-picker__option-credit-info {
 		float: right;
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/subscription-length-picker/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/site-address-changer/style.scss [hunk 43]
```diff
@@ -23,7 +23,7 @@
 }
 
 .site-address-changer__select-icon {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/site-address-changer/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/sharing-preview-pane/style.scss [hunk 44]
```diff
@@ -41,7 +41,7 @@
 	width: 48px;
 	height: 49px;
 	cursor: pointer;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/sharing-preview-pane/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-visit-link/style.scss [hunk 45]
```diff
@@ -3,7 +3,7 @@
 }
 
 .reader-visit-link__label {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-visit-link/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-subscription-list-item/style.scss [hunk 46]
```diff
@@ -72,7 +72,7 @@
 .reader-subscription-list-item__site-url,
 .reader-subscription-list-item__site-url:visited,
 .reader-subscription-list-item__timestamp {
-	color: var( --color-neutral-200 );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-subscription-list-item/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-site-notification-settings/style.scss [hunk 47]
```diff
@@ -70,7 +70,7 @@
 
 .reader-site-notification-settings__popout-hint,
 .reader-site-notification-settings__popout-instructions-hint {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-site-notification-settings/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-site-notification-settings/style.scss [hunk 48]
```diff
@@ -1,5 +1,5 @@
 .reader-site-notification-settings__button {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-site-notification-settings/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-share/style.scss [hunk 49]
```diff
@@ -1,7 +1,7 @@
 .reader-share__button {
 	align-items: center;
 	box-sizing: border-box;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-share/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-related-card/style.scss [hunk 50]
```diff
@@ -1,5 +1,5 @@
 .reader-related-card__heading {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-related-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-recommended-sites/style.scss [hunk 51]
```diff
@@ -5,7 +5,7 @@
 }
 
 .reader-recommended-sites__header {
-	color: var( --color-neutral-200 );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-recommended-sites/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-post-options-menu/style.scss [hunk 52]
```diff
@@ -1,7 +1,7 @@
 .reader-post-options-menu,
 .reader-post-options-menu .ellipsis-menu {
 	.button.is-borderless {
-		color: var( --color-neutral-light );
+		color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-post-options-menu/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-post-card/style.scss [hunk 53]
```diff
@@ -315,7 +315,7 @@
 }
 
 .reader-post-card__tags {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-post-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-post-card/style.scss [hunk 54]
```diff
@@ -266,7 +266,7 @@
 // Need .reader__content to override .reader__content a
 .reader__content .reader-post-card__timestamp-link,
 .reader__content .reader-post-card__tag-link {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-post-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-post-card/style.scss [hunk 55]
```diff
@@ -165,7 +165,7 @@
 		}
 
 		.reader-post-card__timestamp-link {
-			color: var( --color-neutral-light );
+			color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-post-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-import-button/style.scss [hunk 56]
```diff
@@ -19,6 +19,6 @@
 
 .reader-import-button__label {
 	font-size: 14px;
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-import-button/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-full-post/style.scss [hunk 57]
```diff
@@ -527,7 +527,7 @@
 
 .reader-full-post__header-tag-list-item-link,
 .reader-full-post__header-tag-list-item-link:visited {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-full-post/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-full-post/style.scss [hunk 58]
```diff
@@ -508,7 +508,7 @@
 }
 
 .reader-full-post__header-tag-list-item {
-	color: var( --color-neutral-light );
+	color: var( --color-neutral );
```
Comment (drw158) [inline client/blocks/reader-full-post/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/ecommerce-manage-nudge/style.scss [hunk 59]
```diff
@@ -11,7 +11,7 @@
 	position: absolute;
 	top: 14px;
 	right: 18px;
-	color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/ecommerce-manage-nudge/style.scss]:
```suggestion
	color: var( --color-neutral-light );
```

File: client/blocks/image-selector/style.scss [hunk 60]
```diff
@@ -162,10 +162,10 @@
 	width: 24px;
 	height: 24px;
 	margin: 0;
-	color: var( --color-neutral-200 );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/image-selector/style.scss]:
```suggestion
	color: var( --color-neutral-200 );
```

File: client/blocks/image-selector/style.scss [hunk 61]
```diff
@@ -162,10 +162,10 @@
...
 
 	&:hover {
-		color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/image-selector/style.scss]:
```suggestion
	color: var( --color-neutral-light );
```

File: client/blocks/image-selector/style.scss [hunk 62]
```diff
@@ -205,7 +205,7 @@
 
 		&:hover {
 			.gridicon {
-				color: var( --color-neutral-600 );
+				color: var( --color-text );
```
Comment (drw158) [inline client/blocks/image-selector/style.scss]:
```suggestion
				color: var( --color-neutral-600 );
```

File: client/blocks/image-selector/style.scss [hunk 63]
```diff
@@ -282,7 +282,7 @@
 
 	&:hover {
 		.gridicon {
-			color: var( --color-neutral-600 );
+			color: var( --color-text );
```
Comment (drw158) [inline client/blocks/image-selector/style.scss]:
```suggestion
			color: var( --color-neutral-600 );
```

File: client/blocks/login/style.scss [hunk 64]
```diff
@@ -88,7 +88,7 @@
 		a,
 		a:hover {
 			pointer-events: none;
-			color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/login/style.scss]:
```suggestion
		color: var( --color-text-subtle );
```

File: client/blocks/login/style.scss [hunk 65]
```diff
@@ -184,7 +184,7 @@
 
 	span {
 		background: var( --color-white );
-		color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/login/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/post-edit-button/style.scss [hunk 66]
```diff
@@ -27,7 +27,7 @@
 	}
 
 	.post-edit-button__label {
-		color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/post-edit-button/style.scss]:
```suggestion
		color: var( --color-text-subtle );
```

File: client/blocks/post-item/style.scss [hunk 67]
```diff
@@ -46,7 +46,7 @@ $expanded-post-item-outline: 4px solid $post-item-border-color;
 	cursor: pointer;
 
 	&:hover .form-checkbox {
-		border-color: var( --color-neutral-200 );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/post-item/style.scss]:
```suggestion
		border-color: var( --color-neutral-200 );
```

File: client/blocks/post-item/style.scss [hunk 68]
```diff
@@ -97,7 +97,7 @@ a.post-item__title-link:visited {
 	}
 
 	.post-item__panel.is-untitled & {
-		color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/post-item/style.scss]:
```suggestion
		color: var( --color-text-subtle );
```

File: client/blocks/reader-combined-card/style.scss [hunk 69]
```diff
@@ -194,13 +194,13 @@
...
 .reader-combined-card__timestamp-link {
-	color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
 	cursor: pointer;
 
 	&:focus,
 	&:link,
 	&:visited {
-		color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/reader-combined-card/style.scss]:
```suggestion
	color: var( --color-text-subtle );
```

File: client/blocks/reader-post-card/style.scss [hunk 70]
```diff
@@ -165,7 +165,7 @@
 		}
 
 		.reader-post-card__timestamp-link {
-			color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/reader-post-card/style.scss]:
```suggestion
			color: var( --color-text-subtle );
```

File: client/blocks/site-address-changer/style.scss [hunk 71]
```diff
@@ -23,7 +23,7 @@
 }
 
 .site-address-changer__select-icon {
-	color: var( --color-neutral-light );
+	color: var( --color-text-subtle );
```
Comment (drw158) [inline client/blocks/site-address-changer/style.scss]:
```suggestion
	color: var( --color-neutral-light );
```

**Comment by sixhours** [comment]
This LGTM! :shipit:

**Comment by drw158** [comment]
Thanks! Somehow I missed this.

**Pull request merged**